### PR TITLE
First set of fixes to Dockerhub push action.

### PR DESCRIPTION
**Pull request by NlightNFotis**
This PR contains two small fixes to the Dockerhub image push action we wrote:

1. Deactivates the Homebrew PR action temporarily, so that we don't spam it
    everytime we make a new experimental release.
2. Adds a missing `GITHUB_TOKEN` within the scope of the Dockerhub image
    push action so it can have access to metadata required (release version) to
    name the docker image appropriately.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by hannes-steffenhagen-diffblue — APPROVED** [review]
Disabling the homebrew PR isn't a "fix", but yes we should avoid spamming them.

File: .github/workflows/release-packages.yaml
Comment (tautschnig) [inline .github/workflows/release-packages.yaml:121]: Can we please immediately have a PR that will re-enable this (once the PR is merged)? I am _very_ worried about us forgetting about this.

Comment (NlightNFotis) [reply .github/workflows/release-packages.yaml:121]: I can do that immediately then if that eases your mind (even though it's on my personal todo list as a step after getting it working, so I wouldn't forget about it).

I'll submit a new PR reversing this and link to it here.

Comment (NlightNFotis) [reply .github/workflows/release-packages.yaml:121]: Okay the new PR is available at https://github.com/diffblue/cbmc/pull/5748.

Please do not merge it yet - I'll merge it when the dockerhub image push is ready.

Comment (tautschnig) [reply .github/workflows/release-packages.yaml:121]: Thank you @NlightNFotis for creating the follow-up PR!

**Pull request merged**
